lib/services/android-project-service.ts

```diff
@@ -28,13 +29,14 @@ class AndroidProjectService implements IPlatformProjectService {
 	constructor(private $androidEmulatorServices: Mobile.IEmulatorPlatformServices,
 		private $childProcess: IChildProcess,
 		private $errors: IErrors,
-		private $fs: IFileSystem,
+		$fs: IFileSystem,
```

Comment:
as `fs` can be used only in constructor, can you move it below private dependencies